packages/gatsby-source-wordpress/src/models/develop.ts

```diff
@@ -33,7 +22,10 @@ const developStore: IPreviewStore = {
 
       return state
     },
-  } as IDevelopReducers,
-}
+  },
+  effects: () => {
+    return {}
+  },
```

Comment:
There's a bug in the Rematch types that mean that if there are no effects, it infers the type of reducers incorrectly, so we need to add all these empty effects